test/e2e/specs/editor/various/post-title.spec.js

```diff
@@ -0,0 +1,222 @@
+					.set( 'core/edit-post', 'fullscreenMode', false );
+			}, false );
+
+			const pageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			await expect( pageTitleField ).toHaveText(
+				'I am emphasis I am bold I am anchor'
+			);
+
+			// Check the HTML elements have been **rendered** rather than
+			// output in raw form.
+			await expect( pageTitleField.locator( 'css=em' ) ).toHaveText(
+				'emphasis'
+			);
+
+			await expect( pageTitleField.locator( 'css=strong' ) ).toHaveText(
+				'bold'
+			);
+
+			await expect( pageTitleField.locator( 'css=a' ) ).toHaveText(
+				'anchor'
+			);
+		} );
+
+		test( `should show raw HTML in the post title field when in Code view mode `, async ( {
+			page,
+			editor,
+			admin,
+			requestUtils,
+			pageUtils,
+		} ) => {
+			const { id: postId } = await requestUtils.createPost( {
+				title: 'I am <em>emphasis</em> I am <strong>bold</strong> I am <a href="#">anchor</a>',
+				content: 'Hello world',
+				status: 'publish',
+			} );
+
+			await admin.visitAdminPage(
+				'post.php',
+				`post=${ postId }&action=edit`
+			);
+
+			await page.evaluate( () => {
+				window.wp.data
+					.dispatch( 'core/preferences' )
+					.set( 'core/edit-post', 'welcomeGuide', false );
+
+				window.wp.data
+					.dispatch( 'core/preferences' )
+					.set( 'core/edit-post', 'fullscreenMode', false );
+			}, false );
+
+			// switch Editor to code view mode
+			// Open code editor
+			await pageUtils.pressKeys( 'secondary+M' ); // Emulates CTRL+Shift+Alt + M => toggle code editor
+
+			// Check we're in Code view mode.
+			await expect(
+				editor.canvas.getByRole( 'heading', {
+					name: 'Editing code',
+				} )
+			).toBeVisible();
+
+			const codeViewPageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			// Check that the pageTitleField has the raw HTML
+			await expect( codeViewPageTitleField ).toHaveText(
+				'I am <em>emphasis</em> I am <strong>bold</strong> I am <a href="#">anchor</a>'
+			);
+		} );
+
+		test( 'should strip HTML tags when pasting string of HTML into the post title field in Visual mode', async ( {
+			editor,
+			admin,
+			pageUtils,
+		} ) => {
+			await admin.createNewPost();
+
+			const pageTitleField = editor.canvas.getByRole( 'textbox', {
+				name: 'Add title',
+			} );
+
+			await expect( pageTitleField ).toBeFocused();
+
+			pageUtils.setClipboardData( {
+				html: 'I am <em>emphasis</em> I am <strong>bold</strong> I am <a href="#">anchor</a>',
+			} );
+			await pageUtils.pressKeys( 'primary+v' );
+
+			await expect( pageTitleField ).toHaveText(
+				'I am emphasis I am bold I am anchor'
+			);
+
+			// Check the HTML elements have been stripped and are not rendered.
+			await expect( pageTitleField.locator( 'css=em' ) ).toBeHidden();
+
+			await expect( pageTitleField.locator( 'css=strong' ) ).toBeHidden();
+
+			await expect( pageTitleField.locator( 'css=a' ) ).toBeHidden();
```

Comment:
@kevin940726 Is there a better way to assert that the HTML nodes are _not_ present? 

As the Post Title field will now _render_ HTML, I'm looking for these tags in the rendered output (or in this case I'm checking they are _not_ present).

Reply:
Does [`not.toBeAttached()`](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-attached) fit your need here? I think `toBeHidden()` is also valid too 👍 .

Reply:
Apparently toBeAttached isn't available. Maybe a recent addition? 

Maybe I'll stick with hidden. Thanks.